components/tag-list/tag-list-item-mixin.js

```diff
@@ -87,6 +87,7 @@ export const TagListItemMixin = superclass => class extends LocalizeCoreElement(
 				color: var(--d2l-color-ferrite);
 				cursor: pointer;
 				display: flex;
+				line-height: 1rem;
```

Comment:
The tag list items was a case where their heights were determined by the `line-height` and we don't want them to shrink by `2px`.